19_filter_items/index.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="modal-overlay"></div>
+    <!-- header -->
+    <div class="header"><h1>Headphone Shop</h1></div>
+    <!-- filter bottons -->
+    <div class="flex-container">
+      <button class="button all" data-filter="all">All</button>
+      <button class="button headphones" data-filter="headphones">
+        Headphones
+      </button>
+      <button class="button earbuds" data-filter="earbuds">Earbuds</button>
+      <button class="button speakers" data-filter="speakers">Speakers</button>
+    </div>
+    <!-- search bar -->
+    <div class="search">
+      <form role="search">
+        <label for="search-bar" class="visuallyhidden">Search</label>
+        <span class="search-icon"><i class="fas fa-headphones"></i></span>
+        <input
+          type="search"
+          id="search-bar"
+          class="search-bar"
+          placeholder="Search..."
+        />
+      </form>
+    </div>
+
+    <!-- item cards -->
+
+    <ul class="grid-container earbuds">
+      <!-- card 1 -->
+<<<<<<< HEAD
+      <li class="card all earbuds" data-card="0">
+=======
+      <li class="card all earbuds">
+>>>>>>> 13cafcb97b2e1f1a65d458cfc64f06f29ca17797
```

Comment:
I was checking again this PR and noticed that you still have merge conflicts to be fixed. Make sure you fixed all of them ([using VSCode](https://www.youtube.com/watch?v=lXPYqVjyqHM&t=21s)) before committing the changes :)